index.bs

```diff
@@ -1589,43 +1591,43 @@ When an [=XR input source=] |source| for {{XRSession}} |session| has its [=prima
 Transient input {#transient-input}
 ---------------
 
-Some [=/XR device=]s may support <dfn>transient input sources</dfn>, where the [=XR input source=] is only meaningful while performing its [=primary action=]. An example would be mouse, touch, or stylus input against an {{XRSessionMode/"inline"}} {{XRSession}}, which MUST produce a transient {{XRInputSource}} with a {{targetRayMode}} set to {{screen}}. [=Transient input sources=] are only present in the session's [=list of active XR input sources=] for the duration of the the {{selectstart}}, {{select}}, and {{selectend}} event sequence.
+Some [=/XR device=]s may support <dfn>transient input sources</dfn>, where the [=XR input source=] is only meaningful while performing a <dfn>transient action</dfn>, either the [=primary action=] for a [=primary input source=], or a device-specific action for an [=auxiliary input source=]. An example would be mouse, touch, or stylus input against an {{XRSessionMode/"inline"}} {{XRSession}}, which MUST produce a transient {{XRInputSource}} with a {{targetRayMode}} set to {{screen}}, treated as a [=primary action=] for the [[POINTEREVENTS#dfn-primary-pointer]], and as a non-primary [=transient action=] for a non-primary pointer. [=Transient input sources=] are only present in the session's [=list of active XR input sources=] for the duration of the [=transient action=].
 
-[=Transient input sources=] follow a slightly different sequence when firing [=primary action=] events:
+[=Transient input sources=] follow a slightly different sequence when firing [=transient action=] events:
 
 <div class="algorithm" data-algorithm="on-transient-input-start">
 
-When a [=transient input source=] |source| for {{XRSession}} |session| begins its [=primary action=] the UA MUST run the following steps:
+When a [=transient input source=] |source| for {{XRSession}} |session| begins its [=transient action=] the UA MUST run the following steps:
```

Comment:
The text below seems to imply that transient actions can also be primary actions, but this means that select events would be fired twice since this line no longer "replaces" the primary action spec text the way it used to.

Is there ever a case for a non-primary transient action?

Reply:
I've rephrased it in 9d55efa. I wasn't aware that this was a replacement, that's rather subtle...

An example for a non-primary transient action would be adding additional fingers to a multitouch screen, these would create transient input sources and corresponding `inputsourceschange` events, but no `select{,start,end}` events, similar to how non-primary pointer events don't generate compatibility mouse events. I had updated the example in the previous paragraph to cover that.

As a side note, one additional reason for introducing this distinction is that I'm currently working on the DOM Overlay draft spec, where it would be useful to have a concept of transient sources that don't necessarily trigger XR select events, i.e. because they were handled as a DOM event. That's beyond the scope of this PR, and we can revisit this issue as part of that discussion, but I thought that the concept of non-primary transient sources would also be useful on its own.